### DISCUSSION TITLE
When does new GOES data become available?

### DISCUSSION THREAD

**Discussion by apalmer0** [microsoft/PlanetaryComputer#85] [Q&A] [answered]
Apologies if this is documented somewhere already - I'm trying to access recent GOES data, specifically for a window 84-60 hours ago, but I'm not able to fetch anything more recent than June 30 at the moment (~12 days ago, though it seems 6/30 has been a cutoff for a few days now).

Is there documentation anywhere indicating how long it should take before GOES data is available?

Thanks!

**Comment by lossyrob** [comment, marked as answer]
Hi @apalmer0 - thanks for bringing this to our attention. Looks like some of our pipelines were stuck; I've unblocked them and you should see new data in the next couple of hours. Afterwards the GOES data should be updated daily. Apologies for the delay!

**Reply by apalmer0** [reply]
Awesome! Thanks so much for the help and the quick reply!
